repos/re-theme/src/styleInjector/injectHelpers.js

```diff
@@ -122,17 +122,27 @@ export const filterRules = (style, filter) => {
 /**
  * Creates a unique selector based on the passed in className and cssString
  * @function
- * @param {string} className - Original className used as a css selector
+ * @param {string|Array<string>} className - Original className(s) used as a css selector
  * @param {string} cssString - Css rules for the className in string format
- *
+ * @param {string=} filterPrefix - optional prefix to filter by
+ * 
  * @returns {string} - Hashed version of the string
  */
-export const getSelector = (className, cssString) => {
+export const getSelector = (className, cssString, filterPrefix) => {
+
+  // filter by prefix if passed in
+  const filterWithPrefix = cls => {
+    return filterPrefix
+      ? cls.startsWith(filterPrefix)
```

Comment:
Originally the check was only doing this `className.filter(cls => cls)`
  * Which filters out any `falsy` cls values,
  * Which sometimes exist based on how they are passed in from the components

So you should also check here if the class exists, before calling the `startsWith` method
  * Because if the cls value is false, then it will throw an error due to `startsWith` not being a method of `false`

Simple fix to look like this =>
```
    return cls && filterPrefix
      ? cls.startsWith(filterPrefix)
      : cls
```

Reply:
gotcha. updated